.changelog/unreleased/features/ibc-relayer-cli/2310-new-optional-flags-query-connections.md

```diff
@@ -0,0 +1,3 @@
+- New optional flags `--counterparty-chain` and `--verbose` for the command
+  `query connections` ([#2310](https://github.com/informalsystems/ibc-
+  rs/issues/2310))
```

Comment:
Can we remove the newline here? Seems like unclog is still struggling sometimes with not inserting new lines in URLs.